File: MORM.Java/src/MORM/Java/Models/Venda/Estado.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "ESTADO")
public class Estado extends CollectionItem
{
    @CampoAttribute(Campo = "ID_ESTADO", Tipo = CampoTipo.Key)
    public Integer Id_Estado;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "CD_ESTADO", Tipo = CampoTipo.Req)
    public Integer Cd_Estado;
    @CampoAttribute(Campo = "DS_ESTADO", Tipo = CampoTipo.Req)
    public String Ds_Estado;
    @CampoAttribute(Campo = "DS_SIGLA", Tipo = CampoTipo.Req)
    public String Ds_Sigla;
    @CampoAttribute(Campo = "ID_PAIS", Tipo = CampoTipo.Req)
    public Integer Id_Pais;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Estado.cs
using MORM.Util.Atributos;
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("ESTADO")]
    public class Estado : CollectionItem
    {
        [Campo("ID_ESTADO", CampoTipo.Key)]
        public int Id_Estado { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("CD_ESTADO", CampoTipo.Req)]
        public int Cd_Estado { get; set; }
        [Campo("DS_ESTADO", CampoTipo.Req)]
        public string Ds_Estado { get; set; }
        [Campo("DS_SIGLA", CampoTipo.Req)]
        public string Ds_Sigla { get; set; }
        [Campo("ID_PAIS", CampoTipo.Req)]
        public int Id_Pais { get; set; }
    }
}